README.md

```diff
@@ -76,6 +76,8 @@ Options:
                                                  [string] [default: HTTPS_PROXY]
 ```
 
+Notes: The field in a Table cannot be specified to the `fields` option
```

Comment:
In README.md, there is existing note.
I think it's better to unify the prefix `NOTE:` or `Notes:`.
(I prefer `NOTE:` or create small section `Notes` like [this](https://kintone.dev/en/docs/kintone/js-api/events/record-list-event/#save-submit-success-notes))

https://github.com/kintone/cli-kintone/blob/7792d7e9764fad259bdeb632f15ebb84604cceab/README.md?plain=1#L102

Reply:
@mshrtsr 

I tried but I see that the user may be lost the context of notes if we move them to another section.
For example:

### Current
```
#### Upsert records
The field specified as "Key to Bulk Update" should meet the following requirements:

- be Record Number field
- be one of the following field types with "Prohibit duplicate values" enabled
  - Text
  - Number

NOTE: When the field specified as "Key to Bulk Update" is Record Number, the value of the field may have app code of the target app.
```

### After
```
#### Upsert records
The field specified as "Key to Bulk Update" should meet the following requirements:

- be Record Number field
- be one of the following field types with "Prohibit duplicate values" enabled
  - Text
  - Number

#### Notes
- The field in a Table cannot be specified to the `fields` option
- When the field specified as "Key to Bulk Update" is Record Number, the value of the field may have app code of the target app.
```

==> Possibility, the user won't know that the note `When the field specified as "Key to Bulk Update" is Record Number....` is belong to `Upsert records` section.

In [this article](https://kintone.dev/en/docs/kintone/js-api/events/record-list-event/#delete-submit-event), although the same section [Record List Events](https://kintone.dev/en/docs/kintone/js-api/events/record-list-event/#record-list-events), there's a separate note for 
[Save Submit Success Event - app.record.index.edit.submit.success](https://kintone.dev/en/docs/kintone/js-api/events/record-list-event/#save-submit-success-event)
and
 [Delete Submit Event - app.record.index.delete.submit](https://kintone.dev/en/docs/kintone/js-api/events/record-list-event/#delete-submit-event)

So, what do you think?

Reply:
@hung-cybo 
Oh, it's different from what I wanted to mean.

I mean the make h5 section for each section as follows.

```md
                                                                        [string]
      --pfx-file-path        The path to client certificate file        [string]
      --pfx-file-password    The password of client certificate file    [string]
      --proxy                The URL of a proxy server
                                                 [string] [default: HTTPS_PROXY]

##### Notes
- The field in a Table cannot be specified to the `fields` option
```

```md
- be one of the following field types with "Prohibit duplicate values" enabled
  - Text
  - Number

##### Notes
- When the field specified as "Key to Bulk Update" is Record Number, the value of the field may have app code of the target app.
```

Reply:
@mshrtsr 

OIC, I had misunderstood that because there already had an existing one, so we need to unify them in the same section.
Now, I got it. Thank you for your explanation.